README.md

```diff
@@ -18,7 +18,7 @@ pip install bblfsh
 ```bash
 git clone https://github.com/bblfsh/client-python.git
 cd client-python
-make install
+python setup.py install
```

Comment:
I think the "more modern way" would be `pip install .`  but I just checked and this also shows it in pip package list so it's Ok.